### ISSUE TITLE
[Bug] run start_cli_testnet.sh failed on CentOS 7

### ISSUE THREAD

**Issue by winglq** [libra/libra#205]
# 🐛 Bug

## To reproduce

** Code snippet to reproduce **
./scripts/cli/start_cli_testnet.sh

** Stack trace/error message **
```
-- Build files have been written to: /root/packages/libra/target/debug/build/grpcio-sys-c998cb3e25589d95/out/build

--- stderr
CMake Warning at cmake/protobuf.cmake:48 (message):
  gRPC_PROTOBUF_PROVIDER is "module" but PROTOBUF_ROOT_DIR is wrong
Call Stack (most recent call first):
  CMakeLists.txt:140 (include)


CMake Warning at cmake/gflags.cmake:26 (message):
  gRPC_GFLAGS_PROVIDER is "module" but GFLAGS_ROOT_DIR is wrong
Call Stack (most recent call first):
  CMakeLists.txt:142 (include)


CMake Error at CMakeLists.txt:738 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:6171 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:871 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:1329 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:1329 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:7803 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:1715 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:1715 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:8012 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:2694 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:2694 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


CMake Error at CMakeLists.txt:8114 (target_compile_options):
  Error evaluating generator expression:

    $<COMPILE_LANGUAGE:CXX>

  Expression did not evaluate to a known generator expression


thread 'main' panicked at '
command did not execute successfully, got: exit code: 1

build script failed, must exit now', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/cmake-0.1.40/src/lib.rs:832:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed

```

## Expected Behavior

cli started

## System information

**Please complete the following information:**
- 
git cloned
- 
automatically installed by installer
- 
$uname -a
Linux localhost.localdomain 3.10.0-957.5.1.el7.x86_64 #1 SMP Fri Feb 1 14:54:57 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
$cat /etc/redhat-release
CentOS Linux release 7.6.1810 (Core)

**Comment by tim-cotten** [comment]
Your distribution may have an older CMake package. The COMPILE_LANGUAGE:CXX expression requires version >= 3.3

I ran into this on Amazon Linux 2, and was able to solve it by uninstalling the `cmake` package, installing `cmake3`, and creating a symlink /usr/bin/cmake pointing to /usr/bin/cmake3

More details: https://blog.cotten.io/deploying-the-libra-core-blockchain-currency-on-amazon-ec2-f832d4bfadc8

**Comment by winglq** [comment]
thanks. @tim-cotten 

I also find a closed issue about this [#76](https://github.com/libra/libra/issues/76).
